src/core/__tests__/store-data.test.ts

```diff
@@ -103,36 +103,46 @@ describe("storeData", () => {
     await handler(event);
 
     expect(octokit.request).toHaveBeenCalledTimes(2);
-    expect(octokit.request).toHaveBeenNthCalledWith(
-      1,
-      "PUT /repos/{owner}/{repo}/contents/{path}",
-      {
-        committer: { email: "committer_email", name: "committer_name" },
-        content:
-          "eyJkYXRhRXZlbnRTaWduYXR1cmUiOiJ3b3JrZmxvdy1qb2IiLCJtZXRyaWNzU2lnbmF0dXJlIjoid29ya2Zsb3ctam9iIiwiY3JlYXRlZF9hdCI6MTAwLCJvdXRwdXQiOnsiY3JlYXRlZF9hdCI6NDAwMCwic3RhcnRlZF9hdCI6NDAwMCwiY29tcGxldGVkX2F0Ijo1MDAwLCJkdXJhdGlvbiI6MTAwMCwic3RhdHVzIjoiY29tcGxldGVkIiwid29ya2Zsb3dfbmFtZSI6IndvcmtmbG93LW5hbWUiLCJydW5fYXR0ZW1wdCI6MSwic3RlcHMiOltdfSwib3duZXIiOiJvd25lciIsInJlcG8iOiJyZXBvIiwic3RhdHVzIjoic3VjY2VzcyJ9",
-        message: "auto(data): add workflow-job - workflow-job for owner/repo",
-        owner: "deven-org",
-        path: "raw-data/owner/repo/workflow-job/100.json",
-        repo: "telemetry-data",
-        author: {
-          email: "author_email",
-          name: "author_name",
-        },
-      }
-    );
-    expect(octokit.request).toHaveBeenNthCalledWith(
-      2,
-      "PUT /repos/{owner}/{repo}/contents/{path}",
-      {
-        author: { email: "author_email", name: "author_name" },
-        committer: { email: "committer_email", name: "committer_name" },
-        content:
-          "eyJkYXRhRXZlbnRTaWduYXR1cmUiOiJ3b3JrZmxvdy1qb2IiLCJtZXRyaWNzU2lnbmF0dXJlIjoid29ya2Zsb3ctam9iIiwiY3JlYXRlZF9hdCI6MTAwLCJvdXRwdXQiOnsiY3JlYXRlZF9hdCI6NDAwMCwic3RhcnRlZF9hdCI6NDAwMCwiY29tcGxldGVkX2F0Ijo1MDAwLCJkdXJhdGlvbiI6MTAwMCwic3RhdHVzIjoiY29tcGxldGVkIiwid29ya2Zsb3dfbmFtZSI6IndvcmtmbG93LW5hbWUiLCJydW5fYXR0ZW1wdCI6MSwic3RlcHMiOltdfSwib3duZXIiOiJvd25lciIsInJlcG8iOiJyZXBvIiwic3RhdHVzIjoic3VjY2VzcyJ9",
-        message: "auto(data): add workflow-job - workflow-job for owner/repo",
-        owner: "deven-org",
-        path: "raw-data/owner/repo/workflow-job/100.json",
-        repo: "telemetry-data",
-      }
-    );
+    expect((octokit.request as unknown as jest.Mock).mock.calls)
```

Comment:
Small change here to treat this like a snapshot (and be able to update it like other snapshots with `-u`), since it was really annoying to change the encoded content otherwise